cocaine/proxy/proxy.py

```diff
@@ -740,6 +744,7 @@ def stop_condition(body):
                                  message, proxy_error_headers())
 
             except Exception as err:
+                request.logger.exception(err)
```

Comment:
https://docs.python.org/2/library/logging.html#logging.Logger.exception

```python
request.logger.exception("%s: %s", app.id, err)
```
And drop the next line